src/plugins/1/essentials/hooks/index.md

```diff
@@ -29,30 +29,30 @@ this.hooks = {
 
 ```javascript
 module.exports = class HookPlugin {
-  constructor () {}
+  constructor() {}
 
   /*
    Required plugin initialization function
    (see the "Plugin prerequisites" section)
    */
-  init (customConfig, context) {
+  init(customConfig, context) {
     /*
-      Calls the plugin function "myFunction" when the Kuzzle event
-      "document:afterCreate" is triggered after a successful
-      document creation
+      Calls the plugin functions when the Kuzzle events
+      are
      */
     this.hooks = {
-      'document:afterCreate': (...args) => this.myFunction(...args)
+      'core:kuzzleStart': (...args) => this.myFunctionOnStart(...args),
```

Comment:
(nitpicking) We have to provide an arrow function instead of the function reference itself because arrow functions preserve the execution context. But this purpose is somewhat hidden, and hard to spot to anyone without good knowledge of Javascript.

I think that the following is easier to understand and thus preferable in a boilerplate documentation:

```suggestion
      'core:kuzzleStart': this.myFunctionOnStart.bind(this),
```